### PR TITLE
[VUFIND-1697] NPM build:css task should auto-install dependencies

### PR DESCRIPTION
**Pull request by demiankatz**
This is probably not the most elegant solution to this problem, but it works for me...

TODO
- [x] Close [VUFIND-1697](https://openlibraryfoundation.atlassian.net/browse/VUFIND-1697) when merging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,10 +25,11 @@
     "build-dev:less": "grunt lessdev && npm run lessToSass",
     "watch-dev:less": "grunt watch:lessdev",
 
-    "build:scss": "grunt scss",
+    "build:scss-dependencies": "cd themes/bootstrap5 && npm i",
```
Comment (EreMaijala) [inline package.json]: This should have `npm run installBuildDeps` instead of `npm i`.

**Review by EreMaijala — CHANGES_REQUESTED** [review]
I don't like it that this will hit npmjs.org on each run, so I think we need to at least add the `--no-audit` parameter to `npm install` in themes/bootstrap5/package.json. As far as I can see this would avoid hitting the server when everything is up to date.

File: package.json [hunk 2]
```diff
@@ -25,10 +25,11 @@
...
+    "build:scss": "npm run build:scss-dependencies && grunt scss",
     "check:scss": "grunt check:scss",
     "watch:scss": "grunt watch:scss",
```
Comment (EreMaijala) [inline package.json]: Don't we need the the dependency check with these two as well?

**Comment by demiankatz** [comment]
Thanks, @EreMaijala -- I've applied your review suggestions and have also renamed the new `build:scss-dependencies` to `install:scss-dependencies` since the `build` name wasn't really appropriate.

**Review by EreMaijala — APPROVED** [review]
Thanks, seems to work great now! What also nice with the current mechanism is that you can still run grunt directly to avoid going through the setup. Something worth keeping in mind if grunt is phased out at some point!

**Pull request merged**